index.js

```diff
@@ -260,10 +244,26 @@ morgan.token('date', function getDateToken (req, res, format) {
  * response status code
  */
 
-morgan.token('status', function getStatusToken (req, res) {
-  return headersSent(res)
-    ? String(res.statusCode)
-    : undefined
+morgan.token('status', getStatusToken)
+
+/**
+ * colored response status code
+ */
+
+morgan.token('status-colored', function getStatusColoredToken (req, res) {
+  var status = getStatusToken(req, res)
```

Comment:
Shouldn't the status variable here be a number and not a string?

Reply:
yes! I'll update it